Project-18-Food-order-app/src/components/Header/index.jsx

```diff
@@ -0,0 +1,64 @@
+import { useContext, useEffect, useRef, useState } from "react";
+
+import Button from "../UI/Button";
+import Modal from "../UI/Modal";
+import Cart from "../Cart";
+import Checkout from "../Checkout";
+
+import style from "./header.module.scss";
+import logo from "/logo.jpg";
```

Comment:
@sknepali You're importing an image using an absolute path. This might cause issues if the location of the image changes. Consider moving the image to your local assets and import it from there.